src/test/java/SearchTests.java

```diff
@@ -0,0 +1,30 @@
+package tests;
+
+import org.testng.annotations.Test;
+import pageObjects.HomePage;
+import pageObjects.LoginPage;
+import pageObjects.SearchPage;
+public class SearchTests extends BaseTest {
+    LoginPage loginPage;
+    SearchPage searchPage;
+    HomePage homePage;
+    @Test
+    public void addSongToPlaylistViaSearchField() {
+        loginPage = new LoginPage(getThreadLocal());
+        searchPage = new SearchPage(getThreadLocal());
```

Comment:
please check again package and path import the same my commented I put above about Registration Tests